.github/labeler.yml

```diff
@@ -1,6 +1,12 @@
 github_actions:
 - .github/**/*
 
+nginx:
+- conf/nginx/**/*
+
+mongodb:
+- conf/mongodb/create_indexes.js
```

Comment:
you could also add `scripts/refresh_products_tags.js`  ?

Reply:
done